app/static/js/manageVolunteers.js

```diff
@@ -49,32 +49,48 @@ $(document).ready(function() {
     });
 
     function updateSelectVolunteer(){
-      $("#addVolunteerModal input[type=checkbox]").each(function(index, checkbox){
-          if(checkbox["checked"] == true){
-            $("#addVolunteersButton").prop("disabled", false)
-            return false
-          }
-          $("#addVolunteersButton").prop("disabled", true)
-      })
+      let selectedCheckboxes = getSelectedCheckboxes()
+      let buttonContent = $("#addVolunteersButton").html()
+      if (selectedCheckboxes.length > 1) {
+        if ($("#addVolunteersButton").html().charAt(buttonContent.length-1) != "s") {
+          // make the button text plural if there are multiple users selected
+          $("#addVolunteersButton").html(buttonContent + "s")
+        }
+      } else if ($("#addVolunteersButton").html().charAt(buttonContent.length-1) == "s") {
+        // remove the s if it is plural and we have less than 2 volunteers
+        $("#addVolunteersButton").html(buttonContent.slice(0, -1))
+      }
+      // disable the submit button if there are no selectedCheckboxes
+      if (selectedCheckboxes.length <= 0) {
+        $("#addVolunteersButton").prop("disabled", true)
+      } else {
+        $("#addVolunteersButton").prop("disabled", false)
+      }
+
     }
     
+    function getSelectedCheckboxes() {
+      // get all the checkboxes and return a list of users who's
+      // checkboxes are selected
+      let checkboxesDisplayedInModal = $("#addVolunteerModal input[type=checkbox]")
+      let selectedCheckboxes = []
+      $.each(checkboxesDisplayedInModal, function(index, checkbox){
+        if(checkbox["checked"]){
+          selectedCheckboxes.push(checkbox["value"])
+        }
+      })
+      return selectedCheckboxes
+    }
 
   // Adding the new volunteer to the user database table
     $("#addVolunteersButton").click(function(){
         $("#addVolunteersButton").prop("disabled", true)
-        let user = $("#addVolunteerInput").val()
         let eventId = $("#eventID").val()
-        let checkboxlist = $("#addVolunteerModal input[type=checkbox]")
-        let volunteerList = []
-        $.each(checkboxlist, function(index, checkbox){
-            if(checkbox["checked"] == true){
-                volunteerList.push(checkbox["value"])
-            }
-        })
+        let selectedCheckboxes = getSelectedCheckboxes()
         $.ajax({
           url: `/addVolunteersToEvent/${eventId}`,
           type: "POST",
-          data: {"volunteer": volunteerList, "ajax": true},
+          data: {"volunteer": selectedCheckboxes, "ajax": true},
```

Comment:
Not a big deal, but we think that keeping consistency in naming on the client side and server side could enhance readability. In this case, "volunteer" and "selectedCheckboxes" could be named the same